Carmunication/Controllers/MyPage/FriendListViewController.swift

```diff
@@ -8,11 +8,100 @@
+
+    let dummyFriends = ["홍길동", "우니", "배찌", "젠", "레이", "테드", "젤리빈", "김영빈", "피카츄"]
+    private let friendListView = FriendListView()
+
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .systemBackground
 
         navigationItem.title = "친구관리"
+        view.addSubview(friendListView)
+        friendListView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+
+        // 재사용 셀 등록
+        friendListView.friendListTableView.register(
+            FriendListTableViewCell.self,
+            forCellReuseIdentifier: "friendListTableViewCell"
+        )
+        friendListView.friendListTableView.dataSource = self
+        friendListView.friendListTableView.delegate = self
+
+        navigationItem.rightBarButtonItem = UIBarButtonItem(
+            title: "친구추가",
+            style: .plain,
+            target: self,
+            action: #selector(showFriendAddView)
+        )
+        let backButtonItem = UIBarButtonItem(title: " ", style: .plain, target: nil, action: nil)
+        navigationItem.backBarButtonItem = backButtonItem
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        navigationController?.navigationBar.tintColor = UIColor.semantic.accPrimary
+    }
+
+    // [친구추가] 내비게이션 바 버튼 클릭 시 동작
+    @objc private func showFriendAddView() {
+        let friendAddVC = FriendAddViewController()
+        friendAddVC.modalPresentationStyle = .formSheet
+
+        self.present(friendAddVC, animated: true)
+    }
+}
+
+// MARK: - 테이블 뷰 관련 델리게이트 구현
+extension FriendListViewController: UITableViewDataSource, UITableViewDelegate {
+
+    // 섹션 당 셀의 개수
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 1
+    }
+
+    // 섹션의 개수
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return dummyFriends.count
+    }
+
+    // 각 섹션 사이의 간격
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return 0
+    }
+
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return " "
+    }
+
+    // 각 셀의 높이
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 74
+    }
+
+    // 셀 구성
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if let cell = tableView.dequeueReusableCell(
+            withIdentifier: "friendListTableViewCell",
+            for: indexPath
+        ) as? FriendListTableViewCell {
+            // TODO: - 셀에 친구 정보 넣어주기
+            cell.nicknameLabel.text = dummyFriends[indexPath.section]
+            let chevronImage = UIImageView(image: UIImage(systemName: "chevron.right"))
+            chevronImage.tintColor = UIColor.semantic.textBody
+            cell.accessoryView = chevronImage
+            return cell
+        }
+        return UITableViewCell()
```

Comment:
early exit을 잘 활용하시면 코드의 가독성이 올라갈 것 같아요! 😁

https://docs.swift.org/swift-book/documentation/the-swift-programming-language/controlflow/#Early-Exit

```suggestion
        guard let cell = tableView.dequeueReusableCell(
            withIdentifier: "friendListTableViewCell",
            for: indexPath
        ) as? FriendListTableViewCell else {
            return UITableViewCell()
        }
        // TODO: - 셀에 친구 정보 넣어주기
        cell.nicknameLabel.text = dummyFriends[indexPath.section]
        let chevronImage = UIImageView(image: UIImage(systemName: "chevron.right"))
        chevronImage.tintColor = UIColor.semantic.textBody
        cell.accessoryView = chevronImage
        return cell
```

Reply:
습관적으로 if let을 쓰는 경향이 남아있는 것 같네요😅 guard let 구문도 자주 생각해보도록 하겠습니다!